<ui version="4.0" >
 <class>CalDialog</class>
 <widget class="QWidget" name="CalDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>6</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tab_calendar" >
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <property name="elideMode" >
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="localCal" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>734</width>
        <height>501</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Local Calendars</string>
      </attribute>
      <layout class="QGridLayout" >
       <item rowspan="3" row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="sizeConstraint" >
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="lbl_calList" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Shared Calendar List</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QListWidget" name="lst_calList" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>289</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="lbl_calDetail" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Share Details</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QGridLayout" >
         <property name="sizeConstraint" >
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="lbl_name" >
           <property name="text" >
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lne_name" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="lbl_location" >
           <property name="text" >
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <layout class="QGridLayout" >
           <property name="sizeConstraint" >
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item row="0" column="0" >
            <widget class="QLineEdit" name="lne_location" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QPushButton" name="btn_addFile" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>0</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="lbl_status" >
           <property name="text" >
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="cbx_status" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>0</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text" >
             <string>Private</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Public</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="lbl_allow" >
           <property name="text" >
            <string>Allow List:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QListWidget" name="lst_allow" >
           <item>
            <property name="text" >
             <string>&lt;Disabled></string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" >
        <layout class="QHBoxLayout" >
         <property name="sizeConstraint" >
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_remove" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="peerCal" >
      <property name="geometry" >
       <rect>
        <x>26</x>
        <y>45</y>
        <width>760</width>
        <height>509</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Peer Calendars</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
